transfers/transfers.go

```diff
@@ -0,0 +1,67 @@
+package transfers
+
+import "github.com/checkout/checkout-sdk-go/common"
+
+const (
+	transfers = "transfers"
+)
+
+type TransferType string
+
+const (
+	Commission TransferType = "commission"
+	Promotion  TransferType = "promotion"
+	Refund     TransferType = "refund"
+)
+
+// Request
+type (
+	TransferSource struct {
+		Id     string `json:"id,omitempty"`
+		Amount int    `json:"amount,omitempty"`
+	}
+
+	TransferDestination struct {
+		Id string `json:"id,omitempty"`
+	}
+
+	TransferRequest struct {
+		Reference    string              `json:"reference,omitempty"`
+		TransferType TransferType        `json:"transfer_type,omitempty"`
+		Source       TransferSource      `json:"source,omitempty"`
```

Comment:
Same here, we have TransferSource and we return TransferSourceResponse, so I think we need to change this `TransferSource` to `TransferSourceRequest`